tacker/common/utils.py

```diff
@@ -299,3 +300,11 @@ def cpu_count():
         return multiprocessing.cpu_count()
     except NotImplementedError:
         return 1
+
+
+def is_auth_uri_v3(auth_url):
+    return (re.match('.+v3/?$', auth_url) is not None)
```

Comment:
Not sure about this. The key difference btw what you suggest and the upstream is that in the upstream they instanciate the Client class from the correct module and return the object. You suggest to return the whole module as a variable.

I have a feeling that:
1. This must be an anti-pattern
2. This might lead to really weird errors (circular imports, inter-module dependencies etc..) which tbh I don't have the bandwidth right now to debug and fix.

Let's try to get this merged ASAP and this improvement can always be done in a subsequent PR.